README.md

```diff
@@ -166,42 +167,75 @@ Alternatively you are able to use any `F[_]` which is convertible to/from `monix
 The Scala API uses types-conversions for both consumer and producer, that means you don't have to work directly with `Bytes` (however you
 still can, if you want) and you touch only your business class which is then (de)serialized using provided converter.
 
+Since the API is fully effectful don't forget
```

Comment:
This sentence could use some ending ;)

Reply:
I with I know what I wanted to say :smile: